apps/portals/src/SynapseComponent.tsx

```diff
@@ -42,9 +42,8 @@ export const SynapseComponent: React.FC<SynapseComponentWithContextProps> = ({
 }) => {
   // return the synapse object but with token/search params injected into its props from the context created in AppInitializer
   const { props, ...rest } = synapseConfig
-  const key = JSON.stringify(props)
   return (
-    <SynapseContextConsumer key={key}>
```

Comment:
This key should not be necessary. I didn't notice any new issues after removing it and clicking around. It was added in 2019 with no linked issue in the associated PR - https://github.com/Sage-Bionetworks/portals/pull/145.